app/domain/authentication/authn_jwt/consts.rb

```diff
@@ -11,6 +11,10 @@ module AuthnJwt
     IDENTITY_NOT_RETRIEVED_YET = "Identity not retrieved yet"
     URL_IDENTITY_PROVIDER_INTERFACE_NAME = "url-identity-provider"
     TOKEN_IDENTITY_PROVIDER_INTERFACE_NAME = "token-identity-provider"
+    IDENTITY_PATH_RESOURCE_NAME = "identity-path"
```

Comment:
@hilagross what should be the variable name?

Reply:
@tzheleznyak I think that the variable name should stay `identity-path`

Reply:
indeed this is the variable name